catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/EntityRepository.java

```diff
@@ -908,12 +908,12 @@ public EntityUpdater(T original, T updated, Operation operation) {
       this.operation = operation;
     }
 
-    public final void update() throws IOException {
+    public final void update() throws IOException, ParseException {
```

Comment:
May need to make it `throws Exception` in the future. 
Going with this for now.